addons/im_livechat/static/src/legacy/widgets/public_livechat_window/public_livechat_window.js

```diff
@@ -39,9 +39,7 @@ const PublicLivechatWindow = Widget.extend({
     init(parent, messaging, thread, options) {
         this._super(parent);
         this.messaging = messaging;
-        this.options = _.defaults(options || {}, {
-            placeholder: _t("Say something"),
-        });
+        this.options = _.defaults(options || {});
```

Comment:
call to `_defaults` is now useless

Reply:
Not really, `options` could be `undefined`, so this `_.defaults()` makes in an empty object.

Reply:
But actually never the case, so will just remove it

Reply:
Changes done